IdentityCore/src/broker_operation/request/token_request/MSIDBrokerOperationBrowserTokenRequest.m

```diff
@@ -0,0 +1,60 @@
+//
+//  MSIDBrokerOperationBrowserTokenRequest.m
+//  IdentityCore iOS
+//
+//  Created by Rohit Narula on 1/2/20.
+//  Copyright © 2020 Microsoft. All rights reserved.
+//
+
+#import "MSIDBrokerOperationBrowserTokenRequest.h"
+#import "MSIDJsonSerializableTypes.h"
+#import "MSIDAADAuthority.h"
+
+static NSArray *_bundleIdentifierWhiteList = nil;
+
+@implementation MSIDBrokerOperationBrowserTokenRequest
+
+- (instancetype)initWithRequest:(NSURL *)requestURL bundleIdentifier:(NSString *)bundleIdentifier
+{
+    self = [super init];
+    if (self)
+    {
+        _requestURL = requestURL;
+        
+        if (![_bundleIdentifierWhiteList containsObject:bundleIdentifier])
```

Comment:
Shouldn't this logic live on the broker side? Or at least it should be separate from the request model.

Reply:
Addressed. Moved the bundle_identifier list to ADBrokerSSOExtensionRequest.